crates/uv-cli/src/lib.rs

```diff
@@ -2099,14 +2099,42 @@ pub struct InitArgs {
     #[arg(long)]
     pub name: Option<PackageName>,
 
-    /// Create a virtual workspace instead of a project.
+    /// Create a virtual project, rather than a package.
     ///
-    /// A virtual workspace does not define project dependencies and cannot be
-    /// published. Instead, workspace members declare project dependencies.
-    /// Development dependencies may still be declared.
-    #[arg(long)]
+    /// This option is deprecated and will be removed in a future release.
+    #[arg(long, hide = true, conflicts_with = "package")]
     pub r#virtual: bool,
 
+    /// Set up the project to be packaged.
+    ///
+    /// This is the default behavior when using `--lib`.
+    #[arg(long, conflicts_with = "no_package")]
+    pub r#package: bool,
+
+    /// Do not set up the project to be packaged.
+    ///
+    /// This is the default behavior when using `--app`.
+    #[arg(long, conflicts_with = "package", conflicts_with = "lib")]
```

Comment:
`overrides_with`?

Reply:
```suggestion
    #[arg(long, overrides_with = "package", conflicts_with = "lib")]
```

Reply:
Makes sense to me, I think. That's our standard pattern right?